CHANGELOG.md

```diff
@@ -24,6 +24,28 @@ CHANGELOG
 Swift 5.0
 ---------
 
+* [SE-0235][]:
+
+  The standard library now contains a `Result<Success, Failure: Error>` type to 
```

Comment:
```suggestion
  The standard library now contains a `Result` type for manually propagating errors.
```

(It is unnecessary to repeat the generic parameters here when it is presented one line later.)

It is unclear what "manually model" means; all modeling is manual, but what's different here is how errors are propagated. "Error-producing code" sounds to me like code that's ill-formed, producing compiler errors.